src/platform/imx8m/include/platform/platform.h

```diff
@@ -24,7 +24,7 @@ struct timer;
 #define LPSRAM_SIZE 16384
 
 /* IPC Interrupt */
-#define PLATFORM_IPC_INTERRUPT		IRQ_NUM_MU
+#define PLATFORM_IPC_INTERRUPT		7
```

Comment:
Will this come from DT or Zephyr header in the future ?

Reply:
Ideally, src/drivers/imx/ipc.c will be removed and replaced with an IPC mechanism from Zephyr.